src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/AlwaysEncryptedKeyConverter.cs

```diff
@@ -82,10 +35,16 @@ internal static RSAParameters RSAPublicKeyBlobToParams(byte[] keyBlob)
         {
             Debug.Assert(keyBlob.Length == RSAPublicKeyBlob.Size,
                 $"RSA public key blob was not the expected length. Actual: {keyBlob.Length}. Expected: {RSAPublicKeyBlob.Size}");
+
+            byte[] exponent = new byte[RSAPublicKeyBlob.ExponentSize];
+            byte[] modulus = new byte[RSAPublicKeyBlob.ModulusSize];
+            Buffer.BlockCopy(keyBlob, RSAPublicKeyBlob.ExponentOffset, exponent, 0, RSAPublicKeyBlob.ExponentSize);
+            Buffer.BlockCopy(keyBlob, RSAPublicKeyBlob.ModulusSize, modulus, 0, RSAPublicKeyBlob.ModulusSize);
```

Comment:
Shouldn't use `RSAPublicKeyBlob.ModulusOffset` for the second parameter?

Reply:
Yup. Good catch. I think this must have been the point where I'd run out of attention and decided it was time to finish.